akka-actor/src/main/scala/akka/io/Tcp.scala

```diff
@@ -336,9 +336,9 @@ object Tcp extends ExtensionId[TcpExt] with ExtensionIdProvider {
 
   /**
    * Write `count` bytes starting at `position` from file at `filePath` to the connection.
-   * The count must be > 0. The connection actor will reply with a [[CommandFailed]]
-   * message if the write could not be enqueued. If [[WriteCommand#wantsAck]]
-   * returns true, the connection actor will reply with the supplied [[WriteCommand#ack]]
+   * The count must be &gt; 0. The connection actor will reply with a [[CommandFailed]]
```

Comment:
Think we could build in this replacement right into genjavadoc? Seems like this must be applied everywhere, right?